src/main/java/nextstep/fp/Lambda.java

```diff
@@ -27,27 +27,41 @@ public void run() {
     }
 
     public static int sumAll(List<Integer> numbers) {
+        return sumAll(numbers, number -> true);
+    }
+
+    private static int sumAll(List<Integer> numbers, Conditional c){
         int total = 0;
         for (int number : numbers) {
-            total += number;
+            if(c.canDo(number)){
+                total += number;
+            }
         }
         return total;
     }
 
     public static int sumAllEven(List<Integer> numbers) {
+        return sumAllEven(numbers, number -> number % 2 == 0);
```

Comment:
추가로 정의하신 `sumAllEven()`, `sumAllOverThree()` 메서드 모두 `sumAll(List<Integer> numbers, Conditional c)` 메서드와 중복되는 코드들로 보이는데, 추가적인 메서드를 정의할 필요 없이 `sumAll(List<Integer> numbers, Conditional c)`을 호출해도 되지 않을까 생각합니다.
```suggestion
        return sumAll(numbers, number -> number % 2 == 0);
```